orttraining/orttraining/python/training/orttrainer.py

```diff
@@ -254,6 +254,10 @@ def train_step(self, *args, **kwargs):
         input_desc = [*self.model_desc.inputs, self.model_desc.learning_rate]
         output_desc = self.model_desc.outputs
 
+        # Update Learning Rate if Necessary
+        if self.options.lr_scheduler:
```

Comment:
shall it be an error if lr_scheduler in None?

Reply:
If the user doesn't specify a lr scheduler, it will be None. It shouldn't error

Reply:
I guess it is the case of constant lr.